content/reference/sql/version-control/dolt-system-tables.md

```diff
@@ -1342,3 +1346,56 @@ The `violation_info` field is a JSON payload describing the violation.
 
 As with `dolt_conflicts`, delete rows from the corresponding `dolt_constraint_violations` table to signal to dolt that
 you have resolved any such violations before committing.
+
+# Configuration Tables
+
+Configuration Tables can be staged and versioned just like user tables. They always exist, even in an empty database.
+
+## `dolt_ignore`
+
+`dolt_ignore` stores a list of "table name patterns", and a boolean flag for each pattern indicating whether tables that match the patterns should not be staged for commit.
+
+### Schema
+
+```text
++------------+---------+------+-----+
+| Field      | Type    | Null | Key |
++------------+---------+------+-----+
+| pattern    | text    | NO   | PRI |
+| ignored    | tinyint | NO   |     |
++------------+---------+------+-----+
+```
+
+### Notes
+
+The format of patterns is a simplified version of gitignore’s patterns:
+
+- An asterisk "*" matches any number of characters.
+- The character "?" matches any one character.
+- All other characters match exactly.
+
+If a table name matches multiple patterns with different values for `ignored`, the most specific pattern is chosen (a pattern A is more specific than a pattern B if all names that match A also match pattern B, but not vice versa.) If no pattern is most specific, then attempting to stage that table will result in an error.
+
+Tables that match patterns in `dolt_ignore` can be force-committed by passing the `--force` flag to `dolt add` or `CALL dolt_add`.
+
```

Comment:
(minor) Might be nice to mention the new `--ignored` param for `dolt show` here, too, just to help people discover it. 

Speaking of that new param... it would be good to run `dolt dump-docs` to regen the cli reference docs, too. Search for 'dolt dump-docs' in the README for the `docs` repo and it has a few more instructions. It's pretty straightforward, but feel free to hit me up if you hit any snags.

Reply:
Done and done.